package.json

```diff
@@ -146,8 +146,7 @@
     "del": "^3.0.0",
     "dotenv": "6.0.0",
     "eslint": "5.2.0",
-    "eslint-plugin-cypress": "2.0.1",
```

Comment:
Gets rid of `eslint-plugin-cypress` which was doing nothing more than setting a few globals we can do in a few lines of config.